openstack/compute/v2/extensions/resetstate/requests.go

```diff
@@ -0,0 +1,30 @@
+package resetstate
+
+import (
+	"errors"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+//ResetStatus refers to the states usable in ResetState Action
```

Comment:
In order for `godoc` to work properly, there can't be a blank line between this and `type ResetStatus string`. You can verify this yourself if you do:

```shell
$ cd openstack/compute/v2/extensions/resetstate
$ godoc ./ | less
```

Also, please use a space between `//` and `ResetStatus refers to...`